x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_keys_table.tsx

```diff
@@ -19,26 +19,28 @@ import {
   EuiFilterButton,
   EuiFlexGroup,
   EuiFlexItem,
-  EuiHealth,
   EuiLink,
   EuiSearchBar,
   EuiSpacer,
   EuiText,
-  EuiToolTip,
 } from '@elastic/eui';
 import type { CustomComponentProps } from '@elastic/eui/src/components/search_bar/filters/custom_component_filter';
-import moment from 'moment-timezone';
 import type { FunctionComponent } from 'react';
 import React, { createContext, useContext, useState } from 'react';
 
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
+import type {
+  CategorizedApiKey,
+  CreateAPIKeyResult,
+  QueryApiKeySortOptions,
+} from '@kbn/security-api-key-management';
+import { ApiKeyBadge, ApiKeyStatus, TimeToolTip } from '@kbn/security-api-key-management';
 import { UserAvatar, UserProfilesPopover } from '@kbn/user-profile-components';
 
 import { ApiKeysEmptyPrompt, doesErrorIndicateBadQuery } from './api_keys_empty_prompt';
 import type { AuthenticatedUser } from '../../../../common';
-import type { ApiKey, ApiKeyAggregations, BaseApiKey } from '../../../../common/model';
-import type { CreateAPIKeyResult, QueryApiKeySortOptions } from '../api_keys_api_client';
+import type { ApiKeyAggregations } from '../../../../common/model';
```

Comment:
Fixed in https://github.com/elastic/kibana/pull/186171/commits/817aab70138fa7f803c08a3cbf57137f4a5ea81e

Also moved a few other imports to point at the package instead of common/model